### PR TITLE
Update release_template.md

### PR DESCRIPTION
**Pull request by HanaGemela**

### CODE REVIEW

**Review by michaelstingl — APPROVED** [review]
LGTM

File: .github/release_template.md [hunk 1]
```diff
@@ -67,17 +69,18 @@ __TODO__:
 
 ### Communicate the Availability
 * [ ] Final [Marketing and Communication](https://confluence.owncloud.com/display/OG/Marketing+and+Communication)
-   * [ ] Website links for final release (needed for the following posts)
+   * [ ] QA: Ping marketing to update links on https://owncloud.com/desktop-app/ (provide links from github releases; needed for the following posts)
    * [ ] QA: Central post https://central.owncloud.org/tags/c/news/desktop
    * [ ] QA: Inform on ``#updates`` channel, so that marketing knows about the new release
    * [ ] QA: Inform [packagers](https://confluence.owncloud.com/display/OG/Marketing+and+Communication#MarketingandCommunication-Packagers) - ping @dragotin (openSUSE)
```
Comment (michaelstingl) [inline .github/release_template.md:75]: incorporate @fmoc 's change?

Comment (HanaGemela) [reply .github/release_template.md:75]: The new page in Confluence is not ready yet. It can be separate PR later

Comment (michaelstingl) [inline .github/release_template.md:75]:
```suggestion
   * [ ] QA: Inform [packagers](https://confluence.owncloud.com/x/QYLEAg) - ping @dragotin (openSUSE)
```

**Pull request merged**